share/spice/sales_tax_zip_code/sales_tax_zip_code.js

```diff
@@ -0,0 +1,60 @@
+(function (env) {
+   "use strict";
+   env.ddg_spice_sales_tax_zip_code = function(api_result){
+     // Error checking
+     if (!api_result 
+      || !api_result.header
+      || api_result.header.status !== "SUCCESS"
+      || !api_result.salesTaxInfo) {
+        if (api_result.header && api_result.header.errorMessage) {
+            return Spice.failed(api_result.header.errorMessage); 
+        }          
+        return Spice.failed('sales_tax_zip_code');
+      }
+
+        // Display
+        Spice.add({
+            id: "sales_tax_zip_code",
+            name: "Finance",
+            data: api_result.salesTaxInfo,
+            meta: {
+                //itemType: "Results",
+                //searchTerm: api_result.query,
+                sourceName: "snapCX.io",
+                sourceIcon: false,
+                sourceIconUrl: "http://snapcx.io/favicon.ico",
+                sourceUrl: "https://snapcx.io"
+            },
+            normalize: function(item) {
+                var titleResult, subtitleResult, description;
+                var totalTaxRate, stateTaxRate, countyTaxRate, cityTaxRate, messages;
+
+                totalTaxRate   = (item.totalTaxRate * 100).toFixed(2);
+                stateTaxRate   = (item.stateTaxRate * 100).toFixed(2);
+                countyTaxRate  = (item.countyTaxRate * 100).toFixed(2);
+                cityTaxRate    = (item.cityTaxRate * 100).toFixed(2);
+                messages       = item.messages;
+
+                titleResult = "Total tax is "+totalTaxRate+"%";
```

Comment:
If we could determine the city/state from the zipcode it would be awesome to show that instead.

>**6.00%**
> Sales Tax - Wayne, Pennsylvania